script/Ciel/Day 4/04_01_CIEL04_1.txt

```diff
@@ -242,12 +242,12 @@ Where did the sad face from earlier go, Senpai is chuckling.
 [sha:6fd68861835f0d842397f3ab2204ce9e4b062d8c]{
 -- Page 23, Offset 8243.
 -- 「冗談ですよ。わたしだって遠野くんのお家に直接電話なんかしません。乾くんにお願いして、昨夜遠野くんは乾くんの家に泊まった、という電話をしてもらったんです」
-"I'm just kidding. I wouldn't call Tohno-kun's home directly either. I asked Inui-kun to call and say that Tohno-kun stayed at his house last night."
+"I'm just kidding. Even I wouldn't directly call your family like that. I asked Inui-kun to call and tell them you were staying at his house."
 }
 [sha:13d0159d75ec697842866edc8bdbaa662cd0fd5f]{
 -- Page 24, Offset 8244.
 -- 「良かった……それなら問題はないですけど、先輩、冗談きつすぎですよ。今のは本気で心臓が止まるかと思った」
-"Good ......, then there's no problem, but you're joking too hard, Senpai. That one seriously made me think my heart would stop."
+"Good ......, then there's no problem, but that joke was way too extreme. I seriously thought my heart was gonna stop."
```

Comment:
Good... Then there's no problem. That joke was going too far though; I seriously thought my heart was gonna stop.